Source/RxSwift/toSortedArray.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  toSortedArray.swift
+//  RxSwiftExt
+//
+//  Created by Joan Disho on 17.02.18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+extension Observable where E: Sequence, E.Iterator.Element: Comparable {
+    /**
+     Transforms an observable of comparables into an observable of ordered arrays.
+
+     - parameter ascending: Defined the sorting direction. Defaults to ascending.
+     - returns: The sorted observable.
+    */
+    public func toSortedArray<T>(
+        ascending: Bool = true
+        ) -> Observable<[T]> where E.Iterator.Element == T {
```

Comment:
Can we change the signature to have ascending in-line?  And also make the same Element fix.
e.g.

```swift
public func toSortedArray<T>(ascending: Bool = true)
    -> Observable<[T]> where E.Element == T {
```